ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.19__4.6.yaml

```diff
@@ -0,0 +1,209 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+  openshift_knative-v0.24.0_knative-eventing-src:
+    name: knative-v0.24.0
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.24.0_knative-serving-src:
+    name: knative-v0.24.0
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.24.3_knative-eventing-kafka-src:
+    name: knative-v0.24.3
```

Comment:
shouldn't these be `0.25`-based ones?

Reply:
that'll be updated automatically, I think? I can doublecheck tho.